libr/core/cmd_print.c

```diff
@@ -4178,33 +3968,13 @@ static int cmd_print(void *data, const char *input) {
 				if (len == 0) {
 					break;
 				}
+				use_blocksize = len;
 			}
 		}
-		// XXX - this is necessay b/c radare will automatically
-		// swap flags if arch is x86 and bits == 16 see: __setsegoff in config.c
-
```

Comment:
uhm removed hack?

Reply:
i removed this because if there's really this kind of problem, it should be solved when handling `@a` and `@b` and not here. Since nobody is complaining (i think) about `@a`, probably this hack is not necessary anymore (?)